### DISCUSSION TITLE
Question about the latest release

### DISCUSSION THREAD

**Discussion by Thimeo** [Monika-After-Story/MonikaModDev#8112] [QA - tech support + general] [answered]
Sorry if this is common knowledge, but I saw in the changelog of the last update that you made Monika's hair less red. What's the reason behind that? It seem like a harmless detail but you went ahead and change it, I wondered why.

**Comment by multimokia** [comment, marked as answer]
To quote from earlier today:

> The red in her hair was a remnant of the old filter from act 3. Monika's hair isn't really that red in her standing sprites, this addresses that

Basically this is a correction of the remaining red which was in her hair from when the sprites were separated and recoloured for day sprites.
